### PR TITLE
add deployment options

### PR DESCRIPTION
**Pull request by rhagarty**
Split out deployment options - local, openshift, and cf

### CODE REVIEW

File: doc/source/cf.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ibmcloud cf push
+```
+
+> NOTE: The URL route assigned to your application will be displayed as a result of this command. Note this value, as it will be required to access your app.
+
+To view the application, go to the IBM Cloud route assigned to your app. Typically, this will take the form `https://<app name>.mybluemix.net`.
+
+To view logs, or get overview information about your app, use the IBM Cloud dashboard.
+
+[![return](https://raw.githubusercontent.com/IBM/pattern-utils/master/deploy-buttons/return.png)](https://github.com/IBM/watson-discovery-food-reviews#deployment-options)
```
Comment (markstur) [inline doc/source/cf.md]: I doesn't find a #deployment-options header to jump to (name is different). Also, I prefer making the return go to the next section anyway.  README.md#sample-ui-layout   I think that flows a little better.

File: doc/source/local.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+## 2. Run the application
+
+1. Install [Node.js](https://nodejs.org/en/) runtime or NPM.
+1. Start the app by running `npm install`, followed by `npm start`. If running for the first time, please be patient as the initialization process needs to load 1000 json files into your **Watson Discovery Collection** and may take several minutes.
+1. Access the UI by pointing your browser at `localhost:3000`.
+> Note: `PORT` can be configured in the .env file.
+
+[![return](https://raw.githubusercontent.com/IBM/pattern-utils/master/deploy-buttons/return.png)](https://github.com/IBM/watson-discovery-food-reviews#deployment-options)
```
Comment (markstur) [inline doc/source/local.md]: same comment, no #deployment-options found

File: doc/source/openshift.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* Click `Save`.
+
+Go to the `Applications` tab, choose `Deployments` to view the status of your application.
+
+## 3. Run the application
+
+* From the OpenShift or OKD UI, under `Applications` -> `Routes` you will see your app. Click on the `Hostname`to see your Watson Discovery Food Reviews app in action.
+* Save this URL.
+
+[![return](https://raw.githubusercontent.com/IBM/pattern-utils/master/deploy-buttons/return.png)](https://github.com/IBM/watson-discovery-food-reviews#deployment-options)
```
Comment (markstur) [inline doc/source/openshift.md]: same

File: doc/source/cf.md (hunk 1 above)
Comment (rhagarty) [reply doc/source/cf.md]: agree - will return to sample UI

**Pull request merged**